mailpit/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: mailpit
+description: An email and SMTP testing tool with API for developers
+type: application
+version: 0.2.7
+appVersion: 1.6.18
+dependencies:
+- name: common
+  repository: oci://registry-1.docker.io/bitnamicharts
```

Comment:
No need to have it on our fork ?

Reply:
No it is downloaded locally and added to the index file